routers/private/add_track.py

```
@@ -0,0 +1,150 @@
+async def store_playlist_name(message: Message,state: FSMContext):
+    user_id = get_user_id(message)
+    user_db_id = get_db_user_id(user_id)
+    message_text = get_message_text_safe(message)
+
+    playlist_name = message_text.strip()
+    if not playlist_name:
+        await message.answer("❌ Please provide a valid playlist name.")
+        return
+
+    playlist_db_id = get_playlist_id_by_name(user_db_id,playlist_name)
+    if playlist_db_id == False:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not playlist_db_id:` for false checks

Replace with `not playlist_db_id`

(E712)